container/docker/factory.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
I like it too for the record. Its just that we don't enforce it everywhere.

On Tue, Apr 5, 2016 at 1:26 PM, Tim St. Clair <notifications@github.com>
wrote:

> In container/docker/factory.go
> <https://github.com/google/cadvisor/pull/1190#discussion_r58608544>:
>
> > @@ -27,7 +27,7 @@ import (
> >  	"github.com/google/cadvisor/fs"
> >  	info "github.com/google/cadvisor/info/v1"
> >
> > -	"github.com/fsouza/go-dockerclient"
> > +	docker "github.com/fsouza/go-dockerclient"
>
> It's not necessary, but I think it makes it easier to parse the code (and
> figure out what docker refers to). We do it in some other places, but I
> can revert it if you prefer.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1190/files/d2bb061da0404e9bcc5330d7b3792011af75e6f3..d9c864324b03ef9d62620990d267df00039d77d3#r58608544>
>